2/3_dial_bootstrap.md

```diff
@@ -1,13 +1,26 @@
-* You can dial to a multiadr using libp2p .dial
-Try it yourlsef
-* init multiadr, dail libp2p (some more description about the apis)
+Connecting to a bootstrap node
+==============================
 
-Try it locally
-* open one terminal and run the bootsrap 
-* and from other term run index, you would see the connect log!
+Now that we understand what a bootstrap node is, let's think about how to connect to one from another node.  To do that, we have to learn about how to create connections between libp2p nodes.  In libp2p, there are two ways to create connections.  You either use the `dial` function to create a generic connection with another node or the `dialProtocol` function to create a connection using a specific protocol (e.g. `/chat/1.0.0`).  For now, we're just going to worry about connecting to other peers so let's use the bootstrap node we created in the last section as a remote peer and dial up a connection.  
```

Comment:
not sure if there should a comma after is. It would be nice if can use grammarly to check all the basic language errors

Reply:
I reworded the opening phrase to make it a clearly dependent clause.